Update profilephoto-get (adding a note related to guest user access).md (#9500)
microsoftgraph/microsoft-graph-docs-contrib: api-reference/v1.0/api/profilephoto-get.md

> [!NOTE]
> 
> - Retrieving a user's photo using the Microsoft Graph API is currently not supported in Azure AD B2C tenants.

> > - Guest users will have the ability to view profile photos of users, even when the "Guest user access" settings are configured to the "most restrictive" option in Azure Active Directory External Identities settings.

Reviewer: ```suggestion
> - Guest users can view profile photos of users, even when the "Guest user access" settings are configured to the "most restrictive" option in Azure Active Directory External Identities settings.
```
